docs/_docs/templates.md

```diff
@@ -558,19 +686,27 @@ You can also use this tag to create a link in Markdown as follows:
 {% endraw %}
 ```
 
-### Post URL
+Including {% raw %}{{site.baseurl}}{% endraw %} is optional -- it depends on whether you want the link to be absolute or root-relative.
+
+The path to the post, page, or collection is defined as the path starting from the root directory (where your config file is) to the file, not the path from your existing page to the other page. 
+
+For example, suppose you're creating a link page_a.md stored in pages/folder1/folder2 directory to page_b.md stored in pages/folder1 directory. Your path in the link would not be ../page_b.html. Instead, it would be /pages/folder1/page_b.md.
```

Comment:
```diff
- For example, suppose you're creating a link page_a.md stored in pages/folder1/folder2 directory to page_b.md stored in pages/folder1 directory. Your path in the link would not be ../page_b.html. Instead, it would be /pages/folder1/page_b.md.
+ For example, suppose you're creating a link page_a.md stored in `pages/folder1/folder2` directory to `page_b.md` stored in `pages/folder1` directory. Your path in the link would not be `../page_b.html`. Instead, it would be `/pages/folder1/page_b.md`.
```

Is this too explicit?